### PR TITLE
add stealthburner_leds.cfg

### PR DESCRIPTION
**Pull request by nemgrea**

### CODE REVIEW

File: Firmware/stealthburner_leds.cfg [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+#           Example: add STATUS_LEVELING to the beginning of your QGL gcode macro, and then add STATUS_READY 
+#                    to the end of it to set the logo LED and nozzle LEDs back to the `ready` state.
+#
+#           Example: add STATUS_CLEANING to the beginning of your nozzle-cleaning macro, and then STATUS_READY
+#                    to the end of it to return the LEDs back to `ready` state.
+#
+#     5.  Feel free to change colors of each macro, create new ones if you have a need to.  The macros provided below
+#         are just an example of what is possible.  If you want to try some more complex animations, you will most
+#         likely have to use WLED with Moonraker and a small micro-controller (please see the LED thread for help inside
+#         of the stealthburner_beta channel on Discord).
```
Comment (jtdowney) [inline Firmware/stealthburner_leds.cfg:49]: This reference can probably go given the beta channel is long gone.

Comment (nemgrea) [reply Firmware/stealthburner_leds.cfg]: removed beta reference

**Comment by hartk1213** [comment]
One thing I added to mine which made it just a little bit nicer on boot of the printer was this delayed gcode section to call the status_ready macro 

That way the lights are all one color on boot , similar macro is used on the mini12864 display macros
   
```
##  Set RGB values on boot up for each Neopixel. 
##  Index 1 = logo, Index 2 and 3 = nozzle
[delayed_gcode set_sb_leds_on_boot]
initial_duration: 1
gcode:
        status_ready
```

**Comment by jtdowney** [comment]
> One thing I added to mine which made it just a little bit nicer on boot of the printer was this delayed gcode section to call the status_ready macro
> 
> That way the lights are all one color on boot , similar macro is used on the mini12864 display macros
> 
> ```
> ##  Set RGB values on boot up for each Neopixel. 
> ##  Index 1 = logo, Index 2 and 3 = nozzle
> [delayed_gcode set_sb_leds_on_boot]
> initial_duration: 1
> gcode:
>         status_ready
> ```

I like that idea and would use it, but it seemed like part of the interest in leaving it with the default was identifying color order issues with the various chips. There is a whole section on what to do if you don't see magenta and what it means. Perhaps adding this in a commented out section for when you got the basics down?

**Comment by hartk1213** [comment]
Yeah I was gonna also add to keep it commented out until you have the LEDs setup and showing correctly, then you could uncomment it ..but either way works for me

**Comment by hartk1213** [comment]
> > One thing I added to mine which made it just a little bit nicer on boot of the printer was this delayed gcode section to call the status_ready macro
> > 
> > That way the lights are all one color on boot , similar macro is used on the mini12864 display macros
> > 
> > ```
> > ##  Set RGB values on boot up for each Neopixel. 
> > ##  Index 1 = logo, Index 2 and 3 = nozzle
> > [delayed_gcode set_sb_leds_on_boot]
> > initial_duration: 1
> > gcode:
> >         status_ready
> > ```
> 
> I like that idea and would use it, but it seemed like part of the interest in leaving it with the default was identifying color order issues with the various chips. There is a whole section on what to do if you don't see magenta and what it means. Perhaps adding this in a commented out section for when you got the basics down?

Yeah I was gonna also add to keep it commented out until you have the LEDs setup and showing correctly, then you could uncomment it ..but either way works for me

**Review by KiloQubit — APPROVED** [review]

**Pull request merged**